benches/payout-stakers/src/main.rs

```diff
@@ -59,18 +100,18 @@ fn create_test_validators_and_its_nominators(
         .collect()
 }
 
-pub fn derive_user_account(seed: u64) -> KeyPair {
+pub fn derive_user_account_from_numeric_seed(seed: u32) -> KeyPair {
     trace!("Generating account from numeric seed {}", seed);
     keypair_from_string(&format!("//{}", seed))
 }
 
-fn wait_for_10_eras(
+fn wait_for_eras(
```

Comment:
add parameter specifying how many eras this function waits.

Reply:
okay